docs/index.rst

```diff
@@ -0,0 +1,18 @@
+Welcome to pyamplipi's documentation!
+=====================================
+This is a library that helps control `AmpliPi <https://www.amplipi.com>`_ music players.
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   modules
+
+
+Indices and tables
+==================
+
+* :ref:`genindex`
+* :ref:`modindex`
+* :ref:`search`
```

Comment:
From my testing the search page is useless and doesn't work. The actual sidebar search works decently. No need to link to a useless page. 
```suggestion
```